src/sentry/migrations/0170_actor_introduction.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 1.11.29 on 2021-03-02 21:36
+
+from django.db import migrations, models
+import django.db.models.deletion
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+import sentry.models.actor
```

Comment:
Did Django automatically add this? Surprised it's importing an external file

Reply:
Isn't using imports in migrations like this risky as the model could move/be removed in the future causing the migration to fail.

Reply:
Yeah this was automatically added. It's using the enum to populate the choices. I can definitely just write those in manually though.